cognite_toolkit/_cdf_tk/prototypes/commands/import_.py

```diff
@@ -0,0 +1,216 @@
+            if not overwrite and destination_transformation.exists():
+                self.warn(LowSeverityWarning(f"File already exists at {destination_transformation}. Skipping."))
+                continue
+            destination_transformation.write_text(yaml.safe_dump(transformation))
+            if query_path is not None:
+                (destination_folder / f"{destination_transformation.stem}.sql").write_text(query_path.read_text())
+
+            if schedule is not None:
+                destination_schedule = destination_folder / f"{yaml_file.stem}.Schedule.yaml"
+                destination_schedule.write_text(yaml.safe_dump(schedule))
+            if notifications:
+                destination_notification = destination_folder / f"{yaml_file.stem}.Notification.yaml"
+                destination_notification.write_text(yaml.safe_dump(notifications))
+            if verbose:
+                print(f"Imported {yaml_file} to {destination_folder}.")
+            count_by_resource_type["transformation"] += 1
+            count_by_resource_type["schedule"] += 1 if schedule is not None else 0
+            count_by_resource_type["notification"] += len(notifications)
+
+        print(f"Finished importing from {source} to {destination}.")
+        table = Table(title="Import transformation-cli Summary")
+        table.add_column("Resource Type", justify="right", style="cyan")
+        table.add_column("Count", justify="right", style="magenta")
+        for resource_type, count in count_by_resource_type.items():
+            table.add_row(resource_type, str(count))
+        print(table)
+
+    def _load_file(self, yaml_file: Path) -> dict[str, Any] | None:
+        content = read_yaml_file(yaml_file, expected_output="dict")
+        required_keys = {"externalId", "name", "destination", "query"}
+        if missing_keys := required_keys - content.keys():
+            self.warn(
+                LowSeverityWarning(
+                    f"Missing required keys {missing_keys} in {yaml_file}. Likely not a Transformation manifest. Skipping."
+                )
+            )
+            return None
+        return content
+
+    def _convert_transformation(
+        self, transformation: dict[str, Any], source_file: Path
+    ) -> tuple[dict[str, Any], Path | None]:
+        if "shared" in transformation:
+            transformation["isPublic"] = transformation.pop("shared")
+        if "action" in transformation:
+            transformation["conflictMode"] = transformation.pop("action")
+        if "ignoreNullFields" not in transformation:
+            # This is required by the API, but the transformation-cli sets it to true by default.
+            transformation["ignoreNullFields"] = True
+        query_path: Path | None = None
+        if isinstance(transformation["query"], dict):
+            query = transformation.pop("query")
+            if "file" in query:
+                query_path = Path(query.pop("file"))
```

Comment:
Seems you are writing the SQL file path "toolkit-style" above (L79-80), but then you set the query path to whatever was in the manifest here?